### ISSUE TITLE
Leftover (testing?) files written during VCS writes and testing vars loaded on startup

### ISSUE THREAD

**Issue by durack1** [UV-CDAT/uvcdat#1834]
The git tag is:
```
[duro@ocean]$ printenv | grep UVCDAT
UVCDAT_PROMPT_STRING=v2.5.0-41-g1e4d872
UVCDAT_SETUP_PATH=/usr/local/uvcdat/2016-02-04
```

The output files:
```python
geo.vts
geo.vtu
lonlat.vts
lonlat.vtu
```
Are being written to my working directory when I am making the call:
```python
x = vcs.init()
x.plot(var)
```
I'm also getting the rogue variables loaded during a fresh spyder session:
```python
fnm str 1 /usr/local/uvcdat/2016-02-04/share/uvcdat/sample_data/clt.nc
fnmcurv str 1 /usr/local/uvcdat/2016-02-04/share/uvcdat/sample_data/sampleCurveGrid4.nc
fnmirr str 1 /usr/local/uvcdat/2016-02-04/share/uvcdat/sample_data/sampleGenGrid3.nc
pth str 1 /usr/local/uvcdat/2016-02-04/share/uvcdat/sample_data
s float32 (120,46,72) clt masked_array
```

**Comment by doutriaux1** [comment]
@durack1 thanks the env variable are needed to setup correctly your python (setup_runtime.csh) 
@danlipsa are the `geo.vts` and such left over from you? I'll take a look to locate these files. 
The `fnm` etc come from your .pythonrc or such which you probably got from me :wink:

**Comment by durack1** [comment]
Yep @doutriaux1 no problem with the `env` vars, the VCS and testing files were the bug report

**Comment by danlipsa** [comment]
@doutriaux1 yes, those are from me. My next commit will remove them.

**Comment by durack1** [comment]
And yep @doutriaux1 your `.pythonrc` is the problem with the testing files/vars too:
```python
[duro@ocean]$ more ~/.pythonrc
import readline
# import rlcompleter
readline.parse_and_bind("tab: complete")
import sys
import os
import vtk
renWin = vtk.vtkRenderWindow()
ren = vtk.vtkRenderer()
renWin.AddRenderer(ren)
a = vtk.vtkActor()
m = vtk.vtkPolyDataMapper()

import cdms2
try:
    import cdat_info
    pth = cdat_info.get_sampledata_path()
except:
    pth = sys.prefix
fnm = os.path.join(pth, 'clt.nc')
fnmirr = os.path.join(pth, 'sampleGenGrid3.nc')
fnmcurv = os.path.join(pth, 'sampleCurveGrid4.nc')
f = cdms2.open(fnm)
s = f("clt")
```

**Comment by durack1** [comment]
@danlipsa can you tag this issue as closed with your next commit? a `fix #1834` in the commit message should sort it..

**Comment by danlipsa** [comment]
@durack1 See https://github.com/UV-CDAT/uvcdat/pull/1855 which has just been merged into master.
